tensorboardX/writer.py

```diff
@@ -96,12 +106,6 @@ def add_summary(self, summary, global_step=None):
         event = event_pb2.Event(summary=summary)
         self._add_event(event, global_step)
 
-    def add_graph_onnx(self, graph):
```

Comment:
why move this?

Reply:
Alphabetize methods for better organization - I also alphabetized and organized imports in files.